common-service-core/src/main/java/com/ca/mfaas/eurekaservice/client/util/StringUtils.java

```diff
@@ -7,12 +7,19 @@
  *
  * Copyright Contributors to the Zowe Project.
  */
-package com.ca.mfaas.product.utils;
```

Comment:
Why this class is in eurekaservice package now?

Reply:
Original ...product.utils look to me too specific, but I have to agree that ...eurekaservice.client.utils is not better.

Same for StringUtils.

I'd put them in a new generic package com.ca.mfaas.utils.

Agree?

Reply:
Done.